src/app/shell/personal-cabinet/provider/create-competition/create-competition-description-form/create-competition-description-form.component.spec.ts

```
@@ -0,0 +1,119 @@
+      shortStay: new FormControl(false),
+      isSelfFinanced: new FormControl(false),
+      enrollmentProcedureDescription: new FormControl(''),
+      specialNeedsType: new FormControl('None'),
+      areThereBenefits: new FormControl(false),
+      preferentialTermsOfParticipation: new FormControl(''),
+      educationalShift: new FormControl('First'),
+      ageComposition: new FormControl('SameAge'),
+      coverage: new FormControl('School')
+    });
+    fixture.detectChanges();
+  });
```

Biome (1.9.4)

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)